projects.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: page
+title: Current Projects
+descriptiveTitle: Toronto Mesh Projects
+order: 1
+parent: Get Involved
+---
+
+---
+Node Technology and Mesh Applications
+
+---
+
+### Meshstream
+
+Meshstream is a collection of software and hardware that aims to show what can be accomplished with peer-to-peer applications running over a wireless mesh network that is completely isolated from the Internet.
```

Comment:
Is there a one sentence that encapsulated the multimedia bits below?

Maybe:
"Meshstream is a collection of software and hardware demonstrating peer-to-peer and multimedia applications running over a wireless mesh network that is completely isolated from the Internet."

Reply:
That's better, thanks